### PR TITLE
Test Travis repo

### PR DESCRIPTION
**Pull request by kchitalia**

### CODE REVIEW

File: test/test.xcodeproj/project.xcworkspace/xcshareddata/test.xccheckout [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	<key>IDESourceControlProjectFavoriteDictionaryKey</key>
+	<false/>
+	<key>IDESourceControlProjectIdentifier</key>
+	<string>B21E36CD-CB05-45FA-9EF6-1C5343A797AE</string>
+	<key>IDESourceControlProjectName</key>
+	<string>project</string>
+	<key>IDESourceControlProjectOriginsDictionary</key>
+	<dict>
+		<key>45E40A25E775270CB05800BE8DB589B70D815739</key>
+		<string>https://github.com/kchitalia/test-travis-ci.git</string>
```
Comment (bhariharan) [inline test/test.xcodeproj/project.xcworkspace/xcshareddata/test.xccheckout]: Should be ```https://github.com/forcedotcom/test-travis-ci.git```.

File: test/test.xcodeproj/project.xcworkspace/xcshareddata/test.xccheckout [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	</dict>
+	<key>IDESourceControlProjectPath</key>
+	<string>test/test.xcodeproj/project.xcworkspace</string>
+	<key>IDESourceControlProjectRelativeInstallPathDictionary</key>
+	<dict>
+		<key>45E40A25E775270CB05800BE8DB589B70D815739</key>
+		<string>../../..</string>
+	</dict>
+	<key>IDESourceControlProjectURL</key>
+	<string>https://github.com/kchitalia/test-travis-ci.git</string>
```
Comment (bhariharan) [inline test/test.xcodeproj/project.xcworkspace/xcshareddata/test.xccheckout]: Should be ```https://github.com/forcedotcom/test-travis-ci.git```.

**Comment by bhariharan** [comment]
@kchitalia The ```xcuserdata``` folder should be added to ```.gitignore``` and be removed from this checkin. It should not be checked into source control.

**Comment by kchitalia** [comment]
@bhariharan This is just a test repo, I am going to delete it once I finish testing out some travis CI scripts. :)
Let me see if I can turn off notifications for this.

**Comment by bhariharan** [comment]
If it's a test repo, it should probably be under your user, not in ```forcedotcom``` :-)

**Comment by kchitalia** [comment]
Agreed. Will do that.

**Comment by bhariharan** [comment]
Thanks @kchitalia

**Pull request merged**